.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
```

Comment:
```suggestion
name: TestRelease
```

Reply:
this file can be used to setup an automated release to PyPI as well, although it is not being used as now. If you see the [step-name](https://github.com/Radi85/Comment/pull/228/files/087cea7a81c1642be483141f25d00304c2d04df3#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R32), it does state more explicit information. Do you still want to change this?

Reply:
We agreed that test release only will be used in action pipeline so it makes more sense for me to use `TestRelease`. This can be updated in the future if we go for the main release.